lib_static/plugins/acts_as_attachable/lib/acts_as_attachable.rb

```diff
@@ -134,11 +140,23 @@ module InstanceMethods
         end
 
         class_methods do
+          ##
+          # Can this acts_as_attachable instance accept attachments from the given user
+          # @param user [User]
           def attachments_addable?(user = User.current)
             user.allowed_to_globally?(attachable_options[:add_on_new_permission]) ||
               user.allowed_to_globally?(attachable_options[:add_on_persisted_permission])
           end
 
+          ##
+          # Can this acts_as_attachable instance accept uncontainered attachments from the given user
+          # @param user [User]
+          def uncontainered_attachable?(user = User.current)
+            return false unless attachable_options[:allow_uncontainered]
```

Comment:
Basically I would've expected a `unless containered? || allow_uncontainered?` here.

Reply:
As this is a class method, there is no container we can check for in this case